src/main/ngapp/package-lock.json

```diff
@@ -5629,6 +5629,11 @@
       "integrity": "sha512-tmRv0AVuR7ZyouUHLeNSiO6pqulF7dYa3s19c6t+wz9LD69/uSzdMxJ2S91nTI9U3rt/IldxpzMOFejp6f0hjg==",
       "dev": true
     },
+    "font-awesome": {
+      "version": "4.7.0",
+      "resolved": "https://registry.npmjs.org/font-awesome/-/font-awesome-4.7.0.tgz",
+      "integrity": "sha1-j6jPBBGhoxr9B7BtKQK7n8gVoTM="
+    },
```

Comment:
i dont think you are supposed to add package-lock.json to PRs. since it is always made anew when npm-install is called

Reply:
That's true. Will omit it on the next run

Reply:
Okay